exercises/perfect-numbers/perfect_numbers_test.go

```diff
@@ -19,19 +32,7 @@ func TestGivesPositiveRequiredError(t *testing.T) {
 }
 
 func TestClassifiesCorrectly(t *testing.T) {
-	cases := []struct {
-		input    uint64
-		expected Classification
-	}{
-		{1, ClassificationDeficient},
-		{13, ClassificationDeficient},
-		{12, ClassificationAbundant},
-		{6, ClassificationPerfect},
-		{28, ClassificationPerfect},
-		{496, ClassificationPerfect},
-		{8128, ClassificationPerfect},
-	}
-	for _, c := range cases {
+	for _, c := range testCases {
```

Comment:
Since testcases aren't used by multiple tests, I'm not 100% sure about this change.